docs/using/tutorial/moving-volume-tutorial.rst

```diff
@@ -0,0 +1,184 @@
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :alt: In the initial server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+If you have any feedback or problems, you can :ref:`talk-to-us`.
+
+Before You Begin
+================
+
+Requirements
+------------
+
+To replicate the steps demonstrated in this tutorial, you will need:
+
+* Ubuntu 14.04, Ubuntu 15.04 or OS X with `Homebrew`_ installed.
+* Two Virtual Machines (VMs) on your local machine.
+  For this tutorial, you are supplied with Vagrant images to create the tutorial environment on VMs using VirtualBox, so you must have `Vagrant`_ and `VirtualBox`_ installed.
+* At least 4GB RAM available.
+
+Installing the Client
+=====================
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following:
+
+OS X
+----
+
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+------------
+
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+Ubuntu 15.04
+------------
+
+.. task:: install_cli ubuntu-15.04
+   :prompt: you@laptop:~$
+
+Installing Flocker on Local VMs
+===============================
+
+In this step, you will install two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+
+#. Download the Vagrant configuration file, and the cluster and user credentials:
+
+   .. version-code-block:: console
+
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key
+
+#. Use ``vagrant up`` to start and provision the VMs:
+
+   .. prompt:: bash you@laptop:~$
+
+      vagrant up
+      [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
+      ssh-add ~/.vagrant.d/insecure_private_key
```

Comment:
I don't think this works for Vagrant 1.7 or up, they got rid of the shared "insecure private key". Fine if you're using Vagrant 1.6.x, so the bit about needing Vagrant in the earlier "Requirements" section should probably mention that. I can't remember what the process is for 1.7, you can find the key by running `vagrant ssh-config`